frontend/app/src/views/ImportData.vue

```diff
@@ -178,6 +178,65 @@
             </v-col>
           </v-row>
           <v-divider class="mt-2 mb-4" />
+          <v-row>
+            <v-col cols="12">
+              <div class="import-data__crypto-com">
+                <v-img
+                  max-width="200"
+                  :src="require('@/assets/images/import/shapeshift.svg')"
```

Comment:
Why not use the same image as "shapeshift-fox.svg"?

Reply:
Keeping in line with the UI/UX of the Import Data component.  Other importers use horizontal branding, and shapeshift provides official horizontal branding at https://shapeshift.com/brand-assets.

Reply:
I removed the svg assets in favor of components containing the svg paths.